data-otservbr-global/migrations/36.lua

```diff
@@ -1,3 +1,11 @@
 function onUpdateDatabase()
-	return false -- true = There are others migrations file | false = this is the last migration file
+	logger.info("Updating database to version 37 (add kv_store table)")
+	db.query([[
+		CREATE TABLE kv_store (
+			key_name VARCHAR(191) PRIMARY KEY,
```

Comment:
191 sounds arbitrary, should we at least use a ^2 value? Like 128 or 256?

Reply:
It does doesn't it, but it isn't. https://www.grouparoo.com/blog/varchar-191